src/lib/utilities/find-value-if-it-includes-word.ts

```diff
@@ -0,0 +1,8 @@
+export const findValueIfItIncludesWord = (object, word) => {
+    for (var property in object) {
+      if (object.hasOwnProperty(property) && 
+         property.toString().includes(word)) {
+         return object[property];
+      }
+    }
+  }
```

Comment:
No newline on EOF

https://thoughtbot.com/blog/no-newline-at-end-of-file

Talks about why it matters, and also solutions for various editors.